packages/rlc-common/src/interfaces.ts

```diff
@@ -251,6 +251,7 @@ export interface RLCOptions {
   clearOutputFolder?: boolean;
   ignorePropertyNameNormalize?: boolean;
   compatibilityQueryMultiFormat?: boolean;
+  typespecTitleMap?: Record<string, string>;
```

Comment:
could you update the readme with this map? i guess the key is typespec name and the value is the name we'd like to rename?

Reply:
we can't because of this issue https://github.com/Azure/autorest.typescript/issues/2698